packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -1383,6 +1389,79 @@ export const getUserChallenges = (
   return doGet(url, accessToken, BackendDestinationEnum.REPO_ENDPOINT)
 }
 
+/**
+ * Get a list of teams registered to the given challenge.
+ * see http://rest-docs.synapse.org/rest/GET/challenge.html
+ */
+export const getChallengeTeams = (
+  accessToken: string | undefined,
+  challengeId: string | number,
+  offset: string | number = 0,
+  limit: string | number = 200,
+): Promise<ChallengeTeamPagedResults> => {
+  const url = `/repo/v1/challenge/${challengeId}/challengeTeam?&offset=${offset}&limit=${limit}`
+  return doGet(url, accessToken, BackendDestinationEnum.REPO_ENDPOINT)
+}
+
+/**
+ * Get a list of all teams registered to the given challenge.
+ * see http://rest-docs.synapse.org/rest/GET/challenge.html
+ */
+export const getAllChallengeTeams = (
+  accessToken: string | undefined,
+  challengeId: string | number,
+  offset: string | number = 0,
+  limit: string | number = 200,
+): Promise<ChallengeTeam[]> => {
+  const url = `/repo/v1/challenge/${challengeId}/challengeTeam?&offset=${offset}&limit=${limit}`
+  const fn = () => {
+    return doGet<PaginatedResults<ChallengeTeam>>(
+      url,
+      accessToken,
+      BackendDestinationEnum.REPO_ENDPOINT,
+    )
+  }
+  return getAllOfPaginatedService(fn)
+}
+
+/**
+ * List the Teams under which the given submitter may submit to the Challenge,
+ * i.e. the Teams on which the user is a member and which are registered for the Challenge.
+ * see https://rest-docs.synapse.org/rest/GET/challenge/challengeId/submissionTeams.html
+ */
+export const getSubmissionTeams = (
+  accessToken: string | undefined,
+  challengeId: string | number,
+  offset: string | number = 0,
+  limit: string | number = 200,
```

Comment:
Again, I think the backend will ignore this limit when it considers it too high (usually supports 20-50).
This should also probably use the getAllOfPaginatedService()

Reply:
The 200 was just me following a pattern I found sprinkled throughout existing calls in SynapseClient. Happy to reduce.

Reply:
I'm only requesting 1 or 2 depending on the context.

Reply:
Reduced default to 50